sklearn/ensemble/tests/test_iforest.py

```diff
@@ -263,14 +263,17 @@ def test_iforest_subsampled_features():
 def test_iforest_average_path_length():
     # It tests non-regression for #8549 which used the wrong formula
     # for average path length, strictly for the integer case
+    # Updated to test non-regression for #11839.
```

Comment:
I would be more explicit here:

`Updated to check average path length when input is < 2 (issue #11839)`